### DISCUSSION TITLE
数学表达式应用不准确（出现BUG）

### DISCUSSION THREAD

**Discussion by k4m00i** [tamlok/vnote#1854] [General]
```
10^3.18=1513
```
我想输入 10 的 3.18次方，但是输出结果不对
![2021-08-26 05-59-20 的屏幕截图.png](https://i.loli.net/2021/08/26/wYCPJT9f3Xx2VUa.png)

**Comment by tamlok** [comment]
可以用一些在线编辑器来学习latex语法：https://latex.codecogs.com/

```
10^{3.18}
```
